k8s/migratev2/values.default.yaml

```diff
@@ -0,0 +1,68 @@
+image:
+  repository: gcr.io/onec-co/datamon-migrator-v2
+  tag: latest
+  pullPolicy: IfNotPresent
+
+repo: ""
+bundle: ""
+label: ""
+newLabel: ""
+context: "dev"
+
+credentials: /home/ubuntu/.config/gcloud/application_default_credentials.json
+#secretName: credentials # <- specify an external secret. If not present a secret is created with the content from the credentials file
+# NOTE: to create an external secret: kubectl create secret generic my-creds --from-file=~/.config/gcloud/application_default_credentials.json
+secret: |
+  CHANGE THIS
+
+# some default config to map as ${HOME}/.datamon/datamon.yaml
+config1:
+  metadata: datamon-meta-data
+  blob: datamon-blob-data
+  credential: /home/project/.config/gcloud/application_default_credentials.json
+
+# some default config to map as ${HOME}/.datamon2/datamon.yaml
+config2:
+  config: workshop-config
+  context: dev
+  credential: /home/project/.config/gcloud/application_default_credentials.json
+
+# the size of the volume claim to attach as download staging
+stagingSize: 2Gi
```

Comment:
might be too small

Reply:
this is just a default value. Will take something higher like 10GB.  What do you suggest. Have to document this parameter too.

Reply:
set default to 10Gi